packages/manager/apps/public-cloud/src/components/easter-eggs/modal/translations/Messages_en_GB.json

```diff
@@ -0,0 +1,8 @@
+{
+  "public_cloud_components_easter_eggs_modal_title": "Enabling Dark Mode (Alpha version)",
+  "public_cloud_components_easter_eggs_modal_paragraph_1": "Well done! You've discovered the key combination that allows you to enable the Dark Mode on your Public Cloud control panel.",
+  "public_cloud_components_easter_eggs_modal_paragraph_2": "You can return to Light Mode at any time by entering the same key combination.",
```

Comment:
Maybe we should propose a simpler option to disable it, no ? ^.^

Reply:
I agree. I will add it to the next iteration where we could possible reach a Beta version or
even better be able to integrate it into the ODS. I will keep you posted.